Cargo.toml

```diff
@@ -195,22 +195,25 @@ http = { version = "0.2.9" }
 #
 # WASM
 #
-wasmtime = { version = "11.0.1", default-features = false, features = ["async", "wat", "jitdump", "parallel-compilation", "cranelift", "pooling-allocator", "component-model"] }
-wasmtime-wasi = "11.0.1"
-wasi-cap-std-sync = "11.0.1"
-wasi-common = "11.0.1"
+wasmtime = { version = "14.0.0", git = "https://github.com/bytecodealliance/wasmtime", features = ["component-model"] }
+wasmtime-wasi = {version = "14.0.0", git = "https://github.com/bytecodealliance/wasmtime" }
+wasi-cap-std-sync = { version = "14.0.0", git = "https://github.com/bytecodealliance/wasmtime" }
+wasi-common = { version = "14.0.0", git = "https://github.com/bytecodealliance/wasmtime" }
 wit-bindgen-core = "0.9.0"
 wit-bindgen-rust = "0.9.0"
 wit-component = "0.11.0"
 
 # Required since the webtransport stuff is yet on crates.io
 # TODO: use crates.io dependencies
 [patch.crates-io]
-web-sys = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
-wasm-bindgen = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
-js-sys = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
-wasm-bindgen-futures = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
-wasm-bindgen-shared = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
+# web-sys = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
+# wasm-bindgen = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
+# js-sys = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
+# wasm-bindgen-futures = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
+# wasm-bindgen-shared = { git = "https://github.com/rustwasm/wasm-bindgen.git" }
+wasmtime = { git = "https://github.com/bytecodealliance/wasmtime" }
+wasmtime-wasi = { git = "https://github.com/bytecodealliance/wasmtime" }
+wasi-common = { version = "14.0.0", git = "https://github.com/bytecodealliance/wasmtime" }
```

Comment:
Maybe not anymore, they were initially, but after forking wasm-bridge I went "might as well go git all the way" since I had to fix up breaking changes in git either way inside wasm-bridge